### PR TITLE
Elissa/prizes

### PR DESCRIPTION
**Pull request by elissaperdue**

### CODE REVIEW

**Comment by MaxLeiter** [comment]
image
On a wide layout can we have it be 3x2?

File: components/front/prizePackages.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	return (
+		<div className={styles.row}>
+			<div className={styles.prizes}>
+				<Fade>
+					{PrizeInfo.map(({ name, color }) => {
+						return (
+							<div className={styles.column} key={name}>
+								<div className={styles.card}>
+									&nbsp;
+									<img alt="" src={`images/prizes/${name}.png`} />
```
Comment (MaxLeiter) [inline components/front/prizePackages.tsx]: We should use the `alt` tag here for accessibility. Can you add a `description` field to each Prize and copy over the text from the image and use that as the alt?

Comment (elissaperdue) [reply components/front/prizePackages.tsx]: yeah cool good point my bad

**Comment by MaxLeiter** [comment]
Awesome! 🎉

**Pull request merged**